notifications/broadcast.py

```diff
@@ -243,7 +239,7 @@ def has_expired(cert_instance, noti_rule_msg):
     relationship = str(noti_rule_msg["relationship"])
 
     subject_list.append("Alert - %s" % (relationship))
-    subject_list.append("\nAn SSL Cert on %s " % \
+    subject_list.append("\tAn SSL Cert on %s " % \
```

Comment:
why are you using lists to format the message?

Reply:
we have a long string(Email message) need to be put together
In my opinion the cleanest way is to have a list of all messages as list items, then use map function to generate the email message